core/applicationstate.cpp

```diff
@@ -0,0 +1,14 @@
+#include "applicationstate.h"
+
+static ApplicationState appState = (ApplicationState)-1; // Set to an invalid value
```

Comment:
space around the `-` ?

Reply:
That's a cast and a unary '-'. We usually don't add space for either of those. This could also be written as:
```
static ApplicationState appState = ApplicationState(-1); // Set to an invalid value
```
Perhaps that's more clear?

Reply:
:faceplant:
Yeah. Right.
Not having the space is the correct thing.